src/Components/Register/Register.js

```diff
@@ -21,59 +21,155 @@ const Register = ({ onSubmit, errorLabel }) => {
+          checkPwdValidity.isPwdMatched = false;
+        };
+      };
     setPayload({
       ...payload,
+      ...checkPwdValidity,
       [name]: value
     });
   };
 
   const handleSubmit = e => {
     e.preventDefault();
-    onSubmit(payload);
+    if ( payload.isPwdMatched ) {
+      onSubmit( payload );
+    } else {
+      alert( "Check password" )
+    };
   };
 
   const { email,
     userName,
     password,
     repeatPassword,
-    alterEgo } = payload,
-  { language }          = config,
-  translations          = getTranslations(language), //ToDo: Replace this language for config directly or from store
-  registerTranslations  = translations.register,
-  titleLabel            = registerTranslations.title,
-  emailLabel            = registerTranslations.email,
-  userNameLabel         = registerTranslations.userName,
-  passwordLabel         = registerTranslations.password,
-  repeatPasswordLabel   = registerTranslations.repeatPassword,
-  alterEgoLabel         = registerTranslations.alterEgo,
-  iconLabel             = registerTranslations.icon,
-  submitLabel           = registerTranslations.submit;
+    alterEgo,
+    isPwdMatched
+  } = payload,
+  { language, passwordSettings }          = config,
+  translations                            = getTranslations(language), //ToDo: Replace this language for config directly or from store
+  registerTranslations                    = translations.register,
+  titleLabel                              = registerTranslations.title,
+  emailLabel                              = registerTranslations.email,
+  userNameLabel                           = registerTranslations.userName,
+  passwordLabel                           = registerTranslations.password,
+  repeatPasswordLabel                     = registerTranslations.repeatPassword,
+  alterEgoLabel                           = registerTranslations.alterEgo,
+  iconLabel                               = registerTranslations.icon,
+  submitLabel                             = registerTranslations.submit,
+  pwdTooltipLabel                         = registerTranslations.passwordTooltip,
+  repeatPwdTooltipLabel                   = registerTranslations.repeatPasswordTooltip,
+  emailTooltipLabel                       = registerTranslations.emailTooltip,
+  userTooltipLabel                        = registerTranslations.userTooltip,
+  alterEgoTooltipLabel                    = registerTranslations.alterEgoTooltip,
+  emailMinSetting                         = passwordSettings.email.minLengthEmail,
+  emailMaxSetting                         = passwordSettings.email.maxLengthEmail,
+  pwdMinSetting                           = passwordSettings.pwd.minLengthPwd,
+  pwdMaxSetting                           = passwordSettings.pwd.maxLengthPwd,
+  userNameMinSetting                      = passwordSettings.userName.minLengthUserName,
+  userNameMaxSetting                      = passwordSettings.userName.maxLengthUserName
+
+  let pwdError = null;
```

Comment:
Remove this and put it back on the ternary if in styles please.

Reply:
Done.